### PR TITLE
A few changes in vic-machine

### LINKED ISSUE #1207: vic-machine output says to set DOCKER_OPTS but it seems to be ignored.
VIC Version:
https://bintray.com/vmware/vic-repo/download_file?file_path=vic_2151.tar.gz

ESX version
VMware-VMvisor-Installer-6.0.0-3634798.x86_64 installed with all defaults in Fusion.

Docker version
(mwilliamson) dubs:~/Downloads/vic 3 $ docker --version
Docker version 1.11.1, build 5604cbe

**Expected behavior:**
vic-machine suggests these settings

INFO[2016-06-24T13:39:26-07:00] DOCKER_HOST=172.16.127.143:2376
INFO[2016-06-24T13:39:26-07:00] DOCKER_OPTS="-H 172.16.127.143:2376 --tls"
INFO[2016-06-24T13:39:26-07:00]
INFO[2016-06-24T13:39:26-07:00] Connect to docker:
INFO[2016-06-24T13:39:26-07:00] docker -H 172.16.127.143:2376 --tls info
INFO[2016-06-24T13:39:26-07:00] Installer completed successfully
 

**Actual behavior:**
When I run docker commands it seems to require that we also add the --tls. That is confusing.

(mwilliamson) dubs:~/Downloads/vic 3 $ env | grep DOCKER
DOCKER_HOST=172.16.127.143:2376
DOCKER_OPTS=-H 172.16.127.143:2376 --tls

(mwilliamson) dubs:~/Downloads/vic 3 $ docker run hello-world
docker: Post http://172.16.127.143:2376/v1.23/containers/create: malformed HTTP response "\x15\x03\x01\x00\x02\x02\x16".
* Are you trying to connect to a TLS-enabled daemon without TLS?.
See 'docker run --help'.

### PR DESCRIPTION
**Pull request by emlin**
Fixes #1207 #1363 

mv generate-cert option to no-tls
change default vch name to virtual-container-host
remove DOCKER_OPTS cause that's not consumed by docker client

### CODE REVIEW

**Comment by andrewtchin** [comment]
lgtm

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by hickeng** [comment]
LGTM

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -39,7 +39,7 @@ Install VIC Appliance To Test Server
 Run VIC Machine Command
     [Tags]  secret
     [Arguments]  ${certs}
-    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --image-datastore=%{TEST_DATASTORE} --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --generate-cert=${certs} --password=%{TEST_PASSWORD} --force=true --bridge-network=network --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --image-datastore=%{TEST_DATASTORE} --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --no-tls=${certs} --password=%{TEST_PASSWORD} --force=true --bridge-network=network --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
```
Comment (caglar10ur) [inline tests/resources/Util.robot]: What does --no-tls=${certs} mean? It should either be there (to tell machine to disable tls support) or should be missing from the parameter list to tell machine to use TLS

Comment (emlin) [inline tests/resources/Util.robot]: That does not make difference right now, but this utility provides the ability to configure tls.

**Comment by emlin** [comment]
♻️

File: tests/resources/Util.robot (hunk 1 above)
Comment (caglar10ur) [inline tests/resources/Util.robot]: I know it's bit late but I suggest passing --no-tls as the argument inside the Util.robot instead passing false and using it. --no-tls=true|false is just confusing. Eg;

```
[Arguments]  ${tls}
${output}=  Run  bin/vic-machine-linux create .... ${tls} ....
```

**Pull request merged**